### PR TITLE
e2e: releases nightly

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

- run nightly on release branch v0.34.x

after 0.35 then we can create a script to fetch the two most recent release branches. 

Closes: #XXX

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    timeout-minutes: 5
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.15"
+      - uses: actions/checkout@v2
+        with: 
+          fetch-depth: 0
+      - run: |
+         git branch -a | grep '^v0.*.x' | tac | head -2 > output
```
Comment (tessr) [inline .github/workflows/test.yml:21]: For now, the only release branch that supports e2e tests is v0.34.x - they weren't written for <= v0.33

**Review by tessr — APPROVED** [review]

**Review by alexanderbez — APPROVED** [review]

**Comment by tessr** [comment]
@marbar3778 heads up that "automerge" doesn't work on workflow-modifying PRs

**Comment by marbar3778** [comment]
Oh, forgot about that. Thank you for the merge

**Comment by tessr** [comment]
Happy to play the role of merge bot once in a while 😁

**Pull request merged**